config/v1alpha2/doc.go

```diff
@@ -0,0 +1,8 @@
+// +k8s:deepcopy-gen=package,register
+// +k8s:defaulter-gen=TypeMeta
+// +k8s:openapi-gen=true
+
+// +kubebuilder:validation:Optional
```

Comment:
This is generally bad practice, please drop the package level setting of default optional vs required